rust/src/http2/http2.rs

```diff
@@ -1282,6 +1282,7 @@ impl HTTP2State {
                             None => panic!("no SURICATA_HTTP2_FILE_CONFIG"),
                         }
                     }
+                    sc_app_layer_parser_trigger_raw_stream_reassembly(flow, dir as i32);
```

Comment:
How does this work for a packet which had multiple PDUs/transactions ?

Reply:
If they are parsed separately in this path, this should happen on completion of each request/response irrespective of the direction or the transaction.
Does this look like the right place to do something like that to you?

Reply:
I do not know and I did not want to dive into the code of `sc_app_layer_parser_trigger_raw_stream_reassembly` 

Does it make sense to call it multiple times for one packet ?
I guess not from its arguments...
I see it is convenient, but does it cost ?

Reply:
> I see it is convenient, but does it cost ?

I had asked the same question to Victor before starting the task. I believe we'd reached the conclusion that the benefit is more than the small cost if there's any. Right now the detection engine is buggy by not inspecting the data every time a request/response completes. More info is in the linked tickets.

Reply:
It just sets a flag, so I think the cost is acceptable.

Reply:
Thanks Victor